fedimintd/src/fedimintd.rs

```diff
@@ -32,9 +33,11 @@ use crate::attach_default_module_init_params;
 /// Time we will wait before forcefully shutting down tasks
 const SHUTDOWN_TIMEOUT: Duration = Duration::from_secs(10);
 
+/// Default meta value to use during config generation
 pub const FM_EXTRA_DKG_META_VAR: &str = "FM_EXTRA_DKG_META";
 
 #[derive(Parser)]
+/// Struct to define server options
```

Comment:
Style: "Struct" here is redundant, the user can see it's a `struct` both in code and rendered documentation. Just "Server options" is more succinct.